clothcalc_placeholder.js

```diff
@@ -9180,7 +9161,7 @@
                                     e(this)
                                         .find(".cell_3")
                                         .append(
-                                            '<img src="' + Game.forumURL + '/the-west/buttons/lastpost.gif" style="position:absolute;cursor:pointer;margin-left:3px;" onclick="Forum.openThread(' + r + ", " + n + ')"></img>')
+                                            '<img src="https://tomrobert.safe-ws.de/lastpost.png" style="position:absolute;cursor:pointer;margin-left:3px;" onclick="Forum.openThread(' + r + ", " + n + ')"></img>')
```

Comment:
I'd suggest embedding the image to the script, like we do with many other (see the beginning of the file). You can use a service like http://b64.io/ to get the code.